### DISCUSSION TITLE
求助：QX配置Universal和 iRingo: 📺 TV双语问题

### DISCUSSION THREAD

**Discussion by wtqsns** [DualSubs/.github#8] [Q&A 问答] [answered]
QX配置Universal和 iRingo: 📺 TV，同时也启用了boxjs ，iPad 端上流媒体（如迪斯尼等）已经可以正常双语翻译了，但tvOS 端所有流媒体始终无法实现双语翻译，求助前辈解决一下！

**Comment by VirgilClyne** [comment, marked as answer]
QX 的 tvOS 版无`脚本`功能

**Reply by wtqsns** [reply]
麻烦问一下前辈，如何增加这个脚本功能，谢谢🙏

**Reply by VirgilClyne** [reply]
别用 QX

**Comment by wtqsns** [comment]
谢谢你的解答！请指教一下，我如何解决这个问题

脆薯饼 ***@***.***>于2024年12月27日 周五15:24写道：

**Comment by wtqsns** [comment]
已经知道了，谢谢你🙏

脆薯饼 ***@***.***>于2024年12月27日 周五16:13写道：
